ros_buildfarm/templates/ci/ci_job.xml.em

```diff
@@ -420,7 +420,9 @@ parameters = [
     'archive_artifacts',
     artifacts=[
       'ros%d-%s-linux-%s-%s-ci.tar.bz2' % (ros_version, rosdistro_name, os_code_name, arch),
-    ] + [image for images in show_images.values() for image in images],
+    ] + [path for path in archive_files] + [
+      image for images in show_images.values() for image in images
+    ],
```

Comment:
Since `archive_files` is always a list, I don't see a reason to use a generator here.
```suggestion
    ] + archive_files + [image for images in show_images.values() for image in images],
```

Reply:
Alright, yeah, I had a brainfart. Fixed in 7efb2d891d8227692e2847c471c260783131f2bb.